06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/output/README.md

```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Heroku
+- **Deliver**: live URLs to a `review` app, your `staging` & `production` apps
+- **With**: *independently*
+
+#### Links
+- [Review](http://tony-invertedindex-review.herokuapp.com/)
```

Comment:
I don't believe this to be a review app. It's supposed to be named according to PRs raised in your repo. Please review this.